oauth2/oauth2_connector.bal

```diff
@@ -227,29 +228,35 @@ returns (boolean) | http:HttpConnectorError {
 }
 
 function <OAuth2Connector oAuth2Connector> getAccessTokenFromRefreshToken (http:Request request)
-                                                        returns (string) | http:HttpConnectorError {
+returns (string) | http:HttpConnectorError {
     http:HttpClient refreshTokenClient = http:createHttpClient(oAuth2Connector.refreshTokenEP,
                                                                oAuth2Connector.clientConfig);
     http:Request refreshTokenRequest = {};
     http:Response httpRefreshTokenResponse = {};
     http:HttpConnectorError connectorError = {};
     boolean useUriParams = oAuth2Connector.useUriParams;
```

Comment:
Wehre do you set useUriParams?

Reply:
The useUriParams needs to be set when initializing the oauth2 config. Refer this https://github.com/TharinduNM/package-scim2/blob/temp/tests/test.bal#L15